tests/spec/invalid.coffee

```diff
@@ -89,3 +89,24 @@ describe "Invalid case tests", ->
       ]).then(cb).catch(
         (err) -> throw err
       ).done (-> done()), done
+
+describe "Print Java error message", ->
+  it "when `validate()` called", (done) ->
+    vnu.validate("<html/>", ("ss": "64"))
+      .catch (err) ->
+        expect(err).to.be.an.instanceof(SyntaxError)
+        expect(err.message).to.match(
+          /^Unexpected token I\nInvalid thread stack size: -Xss64\n/
+        )
+        done()
```

Comment:
In Javascript, every functions are invoked without blocking therefore this invocation is not good IMO because don() might be called before calling `except` functions.